projects/systelab-virtual-keyboard/src/lib/systelab-virtual-keyboard.component.ts

```diff
@@ -221,6 +223,11 @@ export class SystelabVirtualKeyboardComponent implements AfterViewInit {
     }
 
     this.dispatchEvents(button);
+
+    if (this.shiftPressed) {
+      this.toggleShift();
```

Comment:
The name of this method would be better as `toggleLayout()` as what we are toggling is the layout, not the shift.

Reply:
No, we're are toggling Shift. I mean, we're simulating a shift key pulse

Reply:
I changed to toggleShifLayout